kaggle-classification/keras_trainer/model.py

```diff
@@ -98,7 +99,8 @@ def train(self, train):
         batch_size=self.hparams.batch_size,
         epochs=self.hparams.epochs,
         validation_split=0.1,
-        callbacks=callbacks)
+        callbacks=callbacks,
+        verbose=2)
```

Comment:
This is probably a Keras thing I'm not familiar with, but what does `verbose = 2` mean?

Reply:
Added comment